### ISSUE TITLE
Mark traffic destined for relay with correct cluster

### ISSUE THREAD

**Issue by uluyol** [uluyol/heyp-agents#23]
Host-agent is currently not controller the interface that actually talks to AA and WA. We don't care about the other interface.

**Comment by uluyol** [comment]
As of b489683eafd4cfaf246364d7366aa812156ebac2 we handle this correctly, but the problem has evolved.

All the numbers look worse. It seems that we should be doing everything (especially rate limiting) at the envoy machines. After all, rate limiting is best done at the source host. Instead, we're doing it in the relays. We can augment the DCMapper config and flow matching so that the ports on the relay are mapped to AA and WA.

**Comment by uluyol** [comment]
Fixed this with a different solution.

It turns out that the relay applying netem and rate limiting wasn't the problem. The problem was that netem would only buffer 1000 packets (by default) and drop others, so it was effectively rate limiting below what we wanted. I guess this had a different effect when we moved netem onto the relay, but it was a problem nevertheless.

728ede58b15e7b2c4b8914ca477314eceb14915f increases the limit which greatly improves things, and the follow-on commit re-tunes admission control.

Can call this fixed.
